src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "xebia.cakemix.ask-timeout"
```

Comment:
http://docs.scala-lang.org/style/naming-conventions.html still recommends 'java-style' prefixes, but indeed it doesn't seem to be what the cool kids do...